<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddItems</class>
 <widget class="QWidget" name="AddItems">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="2">
    <widget class="QLineEdit" name="lineEditprice"/>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>纸质书</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>虚拟物品</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>服饰</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>电子产品</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEditname"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>商品名称：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>商品类别：</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>商品价格：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>商品描述：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="lineEditnumber"/>
   </item>
   <item row="2" column="2">
    <widget class="QTextEdit" name="textEditintroduce"/>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QPushButton" name="pushButtonadd">
     <property name="text">
      <string>发布商品</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>商品数量：</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
